shared/macros-oval.jinja

```diff
@@ -0,0 +1,64 @@
+{{%- macro oval_lineinfile(msg='', path='', parameter='', value='', default='', application='', rpm_package='') -%}}
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    <metadata>
+      <title>{{{ msg or rule_title }}}</title>
+      <affected family="unix">
+        <platform>multi_platform_all</platform>
```

Comment:
This gets into the whole "what do we want to macro-ize" issue. IMO, it is wrong to claim `multi_platform_all` for every single OVAL we could possibly generate with this. One example would be Debian-like vs. RHEL-like naming on Apache's httpd server: on some systems this is `/etc/apache2`, on others it is `/etc/httpd`.

IMO, this should be configurable. Perhaps we need a macro just for platform expansion (`all` -> `multi_platform_all`, etc.).

Reply:
I agree, I will incorporate this logic when #4596 gets merged.